src/tensors/cpu/backend.h

```diff
@@ -10,25 +10,11 @@ namespace marian {
 namespace cpu {
 
 class Backend : public marian::Backend {
-protected:
-  bool int16_{false};
-  bool int8_{false};
-  bool int8shift_{false};
```

Comment:
Any plans for alternative passing of the "shifted" parameter?

Reply:
Not against it, but can we do that in the next PR? Maybe there are better ways than having it in backend (maybe not).

Reply:
Should we just get rid of the unshifted version?

Reply:
It's slower on pre-VNNI without precomputed Alphas, which we don't have here.

Reply:
From my curiosity, could you let me how much the perf diff is between shifted and unshifted?

Reply:
If the difference isn't dramatic I would be in favor of being pragmatic about this :)